docs/root/operations/cli.rst

```diff
@@ -192,6 +192,21 @@ following are the command line options that Envoy supports.
   more. The administration interface usage is similar. Please see `Administration interface 
   <https://www.envoyproxy.io/docs/envoy/latest/operations/admin>`_ for more detail.
 
+.. option:: --socket-path <path string>
+
+  *(optional)* The output file path to the socket address for:ref:`hot restart <arch_overview_hot_restart>`.
+  Default to "@envoy_domain_socket" which will be created in the abstract namespace. Prefix _{role}_{id}
+  is appended to provide name. All envoy processes wanting to participate in hot-restart together must
+  use the same value for this option.
+
+  **NOTE**: The path started with "@" will be created in the abstract namespace.
+
+.. option:: --socket-mode <string>
+
+  *(optional)* The socket file permission for :ref:`hot restart <arch_overview_hot_restart>`.
+  The range of value is from 000 to 777(Octal number).
```

Comment:
Instead of giving a numeric range, I think it's clearer to say `This must be a valid octal file permission, such as 644. The default value is 600.`

Reply:
Done.